receiver/vcenterreceiver/metadata.yaml

```diff
@@ -99,12 +108,98 @@ attributes:
       - "on"
       - "off"
       - "suspended"
+  entity_status:
+    name_override: status
+    description: The current status of the managed entity.
+    type: string
+    enum:
+      - "red"
+      - "yellow"
+      - "green"
+      - "gray"
   object_name:
     name_override: object
     description: The object on the virtual machine or host that is being reported on.
     type: string
 
 metrics:
+  vcenter.datacenter.cluster.count:
+    enabled: false
+    description: The number of clusters under the datacenter.
+    unit: "{clusters}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation_temporality: cumulative
+    attributes: [entity_status]
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.106.0"
+  vcenter.datacenter.host.count:
+    enabled: false
+    description: The number of hosts under the datacenter.
+    unit: "{hosts}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation_temporality: cumulative
+    attributes: [entity_status, host_power_state]
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.106.0"
+  vcenter.datacenter.vm.count:
+    enabled: false
+    description: The number of VM's under the datacenter.
+    unit: "{virtual_machines}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation_temporality: cumulative
+    attributes: [entity_status, vm_count_power_state]
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.106.0"
+  vcenter.datacenter.datastore.count:
+    enabled: false
+    description: The number of datastores under the datacenter.
+    unit: "{datastores}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation_temporality: cumulative
+    attributes: []
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.106.0"
+  vcenter.datacenter.disk.space:
+    enabled: false
+    description: The amount of space in the datacenter.
+    unit: "By"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation_temporality: cumulative
+    attributes: [disk_state]
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.106.0"
+  vcenter.datacenter.cpu.limit:
+    enabled: false
+    description: The amount of CPU available to the datacenter.
```

Comment:
nit: I know the CPU and Memory descriptions here match the cluster descriptions, but might be good to pick on and standardize a bit.

"The total amount of CPU available to the datacenter."
"The total amount of memory available to the datacenter."

Reply:
resolve